### PR TITLE
Experimental

### PR DESCRIPTION
**Pull request by lilchurro**
Added function for #408, after much mission creep and then pulling back and just adding the dang feature in a very literal and dumb manner. Here are ways to make this less dumb, however: (1) Record in the log that containers and images have been deleted; and (2) Provide a list of containers and images that would be affected prior to asking for confirmation for user to proceed.

### CODE REVIEW

**Comment by lilchurro** [comment]
Sorry this took so dang long, guys.

File: vent/menus/main.py [hunk 1]
```diff
@@ -221,7 +221,20 @@ def popup(original_containers, thr, title):
     def system_commands(self, action):
         """ Perform system commands """
         if action == "reset":
-            # !! TODO
+            okay = npyscreen.notify_ok_cancel("This will remove ALL of Vent's containers and images. Are you sure?",
+                                        title="Confirm system command")
+            if okay:
+                # remove containers
+                d_cli = docker.from_env()
+                list = d_cli.containers.list(filters={'label':'vent'})
```
Comment (cglewis) [inline vent/menus/main.py]: I think you're also going to need `all=True` for containers that are no longer running.

File: vent/menus/main.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 import sys
 import threading
 import time
+import docker
```
Comment (cglewis) [inline vent/menus/main.py]: alphabetical sort, please

**Comment by cglewis** [comment]
per #408, this also needs to remove `~/.vent`

**Comment by lilchurro** [comment]
AAAAHHHHH forgot. Thanks!

**Pull request merged**